install-storageos/020-install-storageos.md

```diff
@@ -0,0 +1,100 @@
+
+First we need to install the StorageOS Operator using the following yaml manifest.
```

Comment:
I'd like to ensure we call out the self-evaluation guide here. These steps are inspired by our self-evaluation guide, please do check it out! etc. etc.